components/config/src/config.rs

```diff
@@ -134,6 +134,10 @@ pub struct Config {
 
     /// Set automatically when instantiating the config. Used for cachebusting
     pub build_timestamp: Option<i64>,
+
+    /// Maximum number of urls in the sitemap.xml file. 
+    /// If number of pages exceed that limit, several sitemaps site are created, referenced in a main sitemap.xml
+    pub sitemap_limit: usize,
```

Comment:
I think it could be hard-coded if you are looking to keep it simple. 
The all point is to comply with Google requirements (50000 urls and max 50Mb), so 30K seems safe enough (With unrealistically long urls I get to 35Mb).

Reply:
It seems unlikely people would want to actually configure that so I would just hardcode it.